R/ts_plot.R

```diff
@@ -11,47 +11,47 @@
 #' @return ggplot Object
 #' @export
 #' @examples
-#' ts_plot("~/Desktop/ts4_jjshares.csv", "earnings", 4)
+#' ts_plot("extdata/example_data.csv", "earnings", 4)
 ts_plot <- function(data, col, frequency){
-  
+
```

Comment:
```suggestion
time <- c("1960 Q1", "1960 Q2")
earnings <- c(0.71, 0.63)
ts_data <- tibble(time, earnings)
ts_plot(data=ts_data, col="earnings")
```
I don't think that an outside user will have access to this .csv file. This example should be executable from the R console by any outside user. 

It would be better to make a toy dataframe of  some fake time series data like in the suggestion. You could add a few more rows to the one I've suggested. That should be sufficient.